modules/userId/index.js

```diff
@@ -409,6 +409,10 @@ function processSubmoduleCallbacks(submodules, cb, priorityMaps) {
 function getIds(priorityMap) {
   return Object.fromEntries(
     Object.entries(priorityMap)
+      .filter(([, getActiveModule]) => {
+        const submodule = getActiveModule();
+        return dep.isAllowed(ACTIVITY_ACCESS_USER_IDS, activityParams(MODULE_TYPE_UID, submodule?.config?.name));
```

Comment:
with this version if an id is provided by two modules A and B, and the preferred source is A, but A was disabled for the group, then you'd get no id. But if the point of the group is to test what happens if I disable module A and keep module B, I think we should use the id from B instead (e.g. perform this check within `getActiveModule`).